toolkits/nature/packages/nature-section-heading/README.md

```diff
@@ -0,0 +1,74 @@
+# Nature Section Heading
+
+[GitHub](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/nature/packages/nature-section-heading) | [NPM](https://www.npmjs.com/package/@springernature/nature-section-heading)
+
+The Nature Section heading is a type of heading used for titles of some subsections of pages on Nature.com.
+
+It uses borders above and below the heading text to make it stand out from other headings on the page.
+
+![Example](https://i.ibb.co/28w2b0S/Screenshot-2021-12-29-at-10-44-13.png)
+
+## When to use this component
+
+Use the Nature Section heading when you want to organise multiple pieces of content into a section on a landing page. For example, a group of cards linking to articles on a similar subject.
+
+## When not to use this component
+
+Don’t use the Nature Section heading unless you’re organising multiple related items of content into a group.
+
+When you need to break up content on article pages into chunks, use [default heading styles](https://frontend-design-system.private.springernature.app/nature/styleguide/typography#headings-nature-journals).
+
+## How it works
+
+Write clear, descriptive headings using sentence case.
+
+
+### Section headings as links
+
+You may want a Section heading to link to another page.
+
+In these cases, include a chevron after the heading text to help users understand that they can click on it.
+
+![Example with chevron](https://i.ibb.co/V3KF5Dy/Screenshot-2021-12-29-at-10-49-24.png)
+
+## Installation
+
+To use the Section heading component, enter the following command in your Terminal:
+
+```
+npm install @springernature/nature-section-heading
+```
+
+Then import the installed component code in your `scss` file:
+
+```scss
+// settings
+@import '@springernature/nature-section-heading/scss/10-settings/section-heading';
+// component
+@import '@springernature/nature-section-heading/scss/50-components/section-heading';
+```
+
+Once you’ve installed the `scss` and JavaScript, build your Section Heading using the following HTML as a guide.
+
+### Section heading
+
+```html
+<h2 class="c-section-heading">Heading Text</h2>
+```
+
+### Section heading as link
+
+```html
+<div class="c-section-heading">
+    <h2>
```

Comment:
I couldn't find any clarity in the [front-end playbook](https://github.com/springernature/frontend-playbook/) documentation for [CSS guidelines](https://github.com/springernature/frontend-playbook/blob/main/css/house-style.md) or [markup guidelines](https://github.com/springernature/frontend-playbook/blob/main/markup/house-style.md) …

Is there a reason why the `h`eading element does not get a BEM'y classname but then the `<a>` tag inside does?

Reply:
Good point. There was no reason really, other than I knew they were the only html elements that the styles could apply to and so it was simple to list them out rather than create a BEM element classname and add that classname to our HTML.

In our CSS Guidelines documentation we do give examples of cases of nesting that are acceptable and we do provide an example for html elements nested inside classnames:

![image](https://user-images.githubusercontent.com/5796370/148222640-ff411c33-584d-4ea7-9f48-fa8940f3e6a8.png)

Do you have any thoughts or experience on what is good practice?

Reply:
I suppose you save on page weight by not including classnames in your html. So potentially that is a benefit, although I don't know how much of a page weight saving that would be overall on a typical page and perhaps it would be negligible.